packages/database/src/schema.prisma

```diff
@@ -1205,3 +1206,26 @@ model Rule6Party {
   //Optional link to AppealCase
   AppealCase AppealCase? @relation(fields: [caseReference], references: [caseReference])
 }
+
+/// Proofs of Evidence, Final Comments, Statements, Planning Obligations and Interested Party Comments
+/// Received from BO will share this model and table
+model Representation {
+  /// this ID is for front office only
+  id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+
+  representationId        String /// The unique identifier for the representation
+  caseId                  Int? /// Internal case identifier
+  caseReference           String     @unique /// External case identifier
```

Comment:
Can this be unique? Can there not be multiple representations for one case?

Reply:
Good spots, got my uniques in the wrong place!